apps/full-site-editing/package.json

```
@@ -131,6 +131,9 @@
 	},
 	"devDependencies": {
 		"@types/wordpress__plugins": "^2.3.6",
-		"@wordpress/eslint-plugin": "^6.0.0"
+		"@babel/preset-typescript": "^7.10.1",
```

Let's match calypso-build:

```suggestion
		"@babel/preset-typescript": "^7.8.3",
```

https://github.com/Automattic/wp-calypso/blob/873f41fe772901ee6669fdf48773a4130867d3d9/packages/calypso-build/package.json#L45